cmd/auth/internal/application/oauth2/server.go

```diff
@@ -57,9 +59,9 @@ func InitServer(manager oauth2.Manager, db *sql.DB, logger golog.Logger, secretK
 	return gServer
 }
 
-func getUserIDByEmail(ctx context.Context, db *sql.DB, email string) (id string, err error) {
-	row := db.QueryRowContext(ctx, `SELECT id FROM users WHERE emailAddress=?`, email)
```

Comment:
password should be hashed, we should also store salt per user

Reply:
Implemented in: 45ebb3a68db284c09d4c262b707b408f2498fd59